leaflet/index.d.ts

```diff
@@ -485,12 +480,26 @@ declare namespace L {
         [name: string]: any;
     }
 
-    export interface TileLayer extends GridLayer {
+    export class TileLayer extends GridLayer {
+        constructor(urlTemplate: string, options?: TileLayerOptions);
         setUrl(url: string, noRedraw?: boolean): this;
+
+        options: TileLayerOptions;
     }
 
     export function tileLayer(urlTemplate: string, options?: TileLayerOptions): TileLayer;
 
+
```

Comment:
This looks good to me. My only request would be if you could remove this extra blank line.